### PR TITLE
updating contrib pages for ref docs

### PR DESCRIPTION
**Pull request by kbhawkey**
See issues #18567 , #14111.

### CODE REVIEW

File: content/en/docs/contribute/generate-ref-docs/kubectl.md [hunk 1]
```diff
@@ -152,65 +135,62 @@ milestone in your pull request. If you don’t have those permissions, you will
...
-create and populate a `v1_15` directory by running these commands:
+In the `<rdocs-base>` directory, run the following build target:
 
 ```shell
-mkdir gen-kubectldocs/generators/v1_15
-cp -r gen-kubectldocs/generators/v1_14/* gen-kubectldocs/generators/v1_15
+make createversiondirs
 ```
 
 ## Checking out a branch in k8s.io/kubernetes
```
Comment (tengqm) [inline content/en/docs/contribute/generate-ref-docs/kubectl.md]: May want to change this to `Checking out a release tag in k8s.io/kubernetes

File: content/en/docs/contribute/generate-ref-docs/kubectl.md [hunk 2]
```diff
@@ -152,65 +135,62 @@ milestone in your pull request. If you don’t have those permissions, you will
...
 
 In your local <k8s-base> repository, checkout the branch that has
 the version of Kubernetes that you want to document. For example, if you want
-to generate docs for Kubernetes 1.15, checkout the release-1.15 branch. Make sure
+to generate docs for Kubernetes 1.17, checkout the v1.17.0 branch. Make sure
```
Comment (tengqm) [inline content/en/docs/contribute/generate-ref-docs/kubectl.md]:
```suggestion
to generate docs for Kubernetes 1.17, checkout the '`v1.17.0`' tag. Make sure
```

**Comment by kbhawkey** [comment]
Page previews:
- https://deploy-preview-18787--kubernetes-io-master-staging.netlify.com/docs/contribute/generate-ref-docs/
- https://deploy-preview-18787--kubernetes-io-master-staging.netlify.com/docs/contribute/generate-ref-docs/contribute-upstream/
- https://deploy-preview-18787--kubernetes-io-master-staging.netlify.com/docs/contribute/generate-ref-docs/quickstart/
- https://deploy-preview-18787--kubernetes-io-master-staging.netlify.com/docs/contribute/generate-ref-docs/kubectl/
- https://deploy-preview-18787--kubernetes-io-master-staging.netlify.com/docs/contribute/generate-ref-docs/kubernetes-components/
- https://deploy-preview-18787--kubernetes-io-master-staging.netlify.com/docs/contribute/generate-ref-docs/kubernetes-api/

File: content/en/docs/contribute/generate-ref-docs/kubectl.md (hunk 1 above)
Comment (kbhawkey) [reply content/en/docs/contribute/generate-ref-docs/kubectl.md]: Good idea

**Comment by kbhawkey** [comment]
Additions:
- a common Prerequisite include file.
- a Quickstart guide describing how to use the update-imported-docs script to build _all_ reference docs
- cleanup of links and instructions where outdated; updated update-imported-docs/README; maintained individual reference generation pages
- brief explanation of the use of versioned directories and tagged k8s/k8s release versions

**Comment by kbhawkey** [comment]
@jimangel , Would you have time over the next week (or before the next release) to review these changes? I am happy to make updates 😃.

**Pull request merged**